### ISSUE TITLE
太多问题了！！！

### ISSUE THREAD

**Issue by Chang-yuanyuan** [FlyMyAI/flymyai-lora-trainer#73]
感谢作者的贡献，但是目前我在使用这份代码的过程中发现了一些问题： 项目中关于qwen_image_edit的lora训练有太多问题了，比如在主函数中qwen_vl需要同时编码图像和文本，但是如果precompute_text_embeddings设置为false时似乎qwen_vl就不会编码图像只编码文本，这就遗漏了条件信息

Thank you for your contribution. However, I found several issues while using the code.

Specifically, there are multiple problems with the LoRA training pipeline for qwen_image_edit. For instance, in the main training loop, qwen_vl is expected to encode both the input image and the text. However, when precompute_text_embeddings is set to false, qwen_vl only processes the text while skipping image encoding, which results in the loss of the image conditioning information.
